db/migrate/20180111143514_create_restaurants.rb

```diff
@@ -0,0 +1,10 @@
+class CreateRestaurants < ActiveRecord::Migration[5.1]
+  def change
+    create_table :restaurants do |t|
+      t.string :name
+      t.text :description
+
```

Comment:
remove this line

Reply:
This file is part of the migration file that is created when you run db:migrate, the layout comes standard with the space.

Reply:
Good, I'm an idiot :)